glean-core/android/metrics.yaml

```diff
@@ -208,39 +208,52 @@ glean.internal.metrics:
     expires: never
 
 glean.error:
- invalid_value:
-   type: labeled_counter
-   description:
-     Counts the number of times a metric was set to an invalid value.
-     The labels are the `category.name` identifier of the metric.
-   bugs:
-     - https://bugzilla.mozilla.org/1499761
-   data_reviews:
-     - https://bugzilla.mozilla.org/show_bug.cgi?id=1499761#c5
-   notification_emails:
-     - telemetry-client-dev@mozilla.com
-   expires: never
-   send_in_pings:
-     - all_pings
-   no_lint:
-     - COMMON_PREFIX
+  invalid_value:
+    type: labeled_counter
+    description:
+      Counts the number of times a metric was set to an invalid value.
+      The labels are the `category.name` identifier of the metric.
+    bugs:
+      - https://bugzilla.mozilla.org/1499761
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1499761#c5
+    notification_emails:
+      - telemetry-client-dev@mozilla.com
+    expires: never
+    send_in_pings:
+      - all_pings
+    no_lint:
+      - COMMON_PREFIX
 
- invalid_label:
-   type: labeled_counter
-   description:
-     Counts the number of times a metric was set with an invalid label.
-     The labels are the `category.name` identifier of the metric.
-   bugs:
-     - https://bugzilla.mozilla.org/1499761
-   data_reviews:
-     - https://bugzilla.mozilla.org/show_bug.cgi?id=1499761#c5
-   notification_emails:
-     - telemetry-client-dev@mozilla.com
-   expires: never
-   send_in_pings:
-     - all_pings
-   no_lint:
-     - COMMON_PREFIX
+  invalid_label:
+    type: labeled_counter
+    description:
+      Counts the number of times a metric was set with an invalid label.
+      The labels are the `category.name` identifier of the metric.
+    bugs:
+      - https://bugzilla.mozilla.org/1499761
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1499761#c5
+    notification_emails:
+      - telemetry-client-dev@mozilla.com
+    expires: never
+    send_in_pings:
+      - all_pings
+    no_lint:
+      - COMMON_PREFIX
+
+  preinit_tasks_timeout:
+    type: boolean
+    description:
+      Set to true if the tasks that are queued prior to Glean initialization
+      time out.
+    bugs:
+      - https://bugzilla.mozilla.org/1575848
+    data_reviews:
+      - TBD
+    notification_emails:
+      - telemetry-client-dev@mozilla.com
+    expires: never
```

Comment:
Are we adding this to investigate a specific problem? If so, do we want to retain this really `forever`?

Reply:
Not to investigate a specific problem.  Just that tasks timing out would lead to data loss, and it would be good to know about it if it ever happens.  I thought `never` because I sort of see this in the same category as the other error reporting -- a general telemetry health metric...

Reply:
That's perfect, I wanted to make sure I wasn't missing something :)